server/server.go

```diff
@@ -309,6 +309,9 @@ func setTxnScope() {
 // Export config-related metrics
 func (s *Server) reportConfig() {
 	metrics.TokenLimitGauge.Set(float64(s.cfg.TokenLimit))
```

Comment:
How about remove the `TokenLimitGauge ` metric and use `ConfigStatus` with the `type="token-limit"`?  @iosmanthus

Reply:
Cool

Reply:
I will create another pull request and cherry-pick this change.